pages/map.js

```diff
@@ -85,14 +87,48 @@ export async function getStaticProps() {
     logger.error(e, "ERROR loading tags");
   }
 
+  try {
+    data.events = await getEvents(true);
+  } catch (e) {
+    logger.error(e, "ERROR loading Events");
+  }
+  
+  data.events = data.events.map((event, index) => {
+    const offset = Math.random() * 0.02; // ~2.2km
+    const offset2 = Math.random() * 0.02; // ~2.2km
+    return {
+      type: "Feature",
+      properties: {
```

Comment:
You need a field that will tell that these items are events when the array is merged with users.

Reply:
Should I edit "type":"Feature" to "type":""user" and "type":"event" Or should I add a new field indicating user and event.

Reply:
This format is a standard geo-coding format.
We can only add fields under properties.

I would suggest adding an `isEvent` field which is set to true on all events.
You don't need to worry about it on users as undefined is false.